apps/website/angular-components/header/README.md

```diff
@@ -43,6 +43,8 @@ If you’re using icons on their own in the header, using tooltips will help use
 <div class="clr-col-sm-12 doc-do">
 <div class="doc-example">
 
+::: inset
```

Comment:
We don't want to uase inset here. THe original demo is using `.doc-wrapper` to set the gray background and container styles. 

The ported angular demos may not have retained their responsiveness from the old site but the way to address this is to use `./apps/website/.vuepress/theme/styles/overrides.scss` for any css needed to override built in behavior that is causing a problem as in inline demo on the site. Keep in mind that in the long term the angular components and demos (from the original docs site) will be removed in favor of the core components and I don't want to make changes to the theme for the Angular components. IMO it is better to use an override and leave a clear `// comment about angular header overside styles` so that it can be easily removed in the future.

Reply:
The issue we are facing here is that the `doc-example` has `::after` selector that adds the `Do` / `Dont` label and if I apply the overflow to the `doc-example` the `Do` / `Dont` label fits into the scrollable area and becomes invisible until you scroll. That's why I need a wrapper on the content so I can make the content wrapper scrollable without hiding the labels

I am up to set the styles in the `overrides.scss` but definitely need a wrapper for the content. Also applying the overflow to the `inset` helps all other examples be scrollable on a tablet view